internal/testutils/helper.go

```diff
@@ -120,4 +99,30 @@ func ValidateImportBlocks(t *testing.T, resource connector.ExportableResource, e
 		}
 		resourceIDs[importBlock.ResourceID] = true
 	}
+
+	expectedImportBlocksMap := map[string]connector.ImportBlock{}
+	for _, importBlock := range *expectedImportBlocks {
```

Comment:
The nil check for this expectedImportBlocks got removed in this change

Reply:
I think range over nil is safe so I removed the check